<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyEventConfigurationForm</class>
 <widget class="QDialog" name="MyEventConfigurationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>189</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Event configurator</string>
  </property>
  <widget class="QFrame" name="frMessageOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>319</width>
     <height>153</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lblAddingPeople">
      <property name="text">
       <string>Event config</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="lblNameInput">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter the event's name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lnNameInput"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="lblDateInput">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Enter the event's date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="datInput"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btnCancel">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btnOk">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
